certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0",
+    "version": "2.0",
     "title": "CyberProof",
     "description": "CyberProof CDC is a next generation incident management and response platform for building Security Operations Centers (SOCs). The platform is a key component of advanced security operations centers - supporting SOC management, orchestration, and incident response. Connect to CyberProof CDC to manage your incidents and communicate with analysts, and automate incident response activities.",
```

Comment:
@Avibek And the description say it is "CyberProof CDC...", Can you please maintain a single title for your connector and remove all the other names or provide detailed explanation

Reply:
@sriyen-msft we already live with this description, is there a special reason we need to change that now?

Reply:
@Avibek Yes, your connector title says "Cyberproof" (earlier it was "Cyberproof Actions"), and the description starts with "Cyberproof CDC.." which creates confusion for the users. It is a standard that all the connector descriptions should be easy to understand and concisely describe the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-1-register-your-connector)

Reply:
@sriyen-msft Fixed
https://github.com/microsoft/PowerPlatformConnectors/pull/1646/commits/a694935b6c683e63651bfd89ce0a249b1c7f994e